lib/beaker/host/unix.rb

```diff
@@ -138,6 +138,23 @@ def external_copy_base
       @external_copy_base
     end
 
+    # Tells you whether a host platform supports beaker's
+    #   {Beaker::HostPrebuiltSteps#set_env} method
+    #
+    # @return [String,nil] Reason message if set_env is not supported,
+    #   nil if it is supported
+    def supports_set_env
```

Comment:
looking at this this morning, I'm realizing your suggested name is much clearer.

I've updated the name.  Thanks for standing up to the wall of text @justinstoller !